packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/OnLayoutEvent.java

```diff
@@ -65,10 +65,10 @@ public String getEventName() {
   @Override
   protected WritableMap getEventData() {
     WritableMap layout = Arguments.createMap();
-    layout.putDouble("x", PixelUtil.toDIPFromPixel(mX));
-    layout.putDouble("y", PixelUtil.toDIPFromPixel(mY));
-    layout.putDouble("width", PixelUtil.toDIPFromPixel(mWidth));
-    layout.putDouble("height", PixelUtil.toDIPFromPixel(mHeight));
+    layout.putDouble("x", PixelUtil.toDIPFromPixel((double) mX));
+    layout.putDouble("y", PixelUtil.toDIPFromPixel((double) mY));
+    layout.putDouble("width", PixelUtil.toDIPFromPixel((double) mWidth));
+    layout.putDouble("height", PixelUtil.toDIPFromPixel((double) mHeight));
```

Comment:
Yoga results are originally in floating point DIPs, so, before we got here, we have converted those to integer pixels that this stores.

Instead of the repeated conversions, and trying to remove the error there, if possible it would be great to use the more original values.

I believe this is closer to how it works under the new architecture. It would be great to test there as well, to see what the results are.